assets/highlight.js

```diff
@@ -1,10 +1,21 @@
-
-$(function(){
-  $('code').each(function(){
-    $(this).html(highlight($(this).text()));
-  });
```

Comment:
I've removed the jQuery dependency too. Makes no sense to load jQuery simply for `$('code').each` and `$(document).ready`.